### PR TITLE
Appamor Profiles for Electrum-NMC binaries

### PR DESCRIPTION
**Pull request by robertmin1**
Appamor Profiles for Electrum-NMC binaries, based on upstream Electrum

### CODE REVIEW

File: contrib/apparmor/usr.local.bin.electrum-nmc [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+include <tunables/global>
+
+@{electrum_exec_path} = /{usr/,usr/local/,*/*/.local/,}bin/electrum-nmc
+
+profile electrum @{electrum_exec_path} {
+  include <abstractions/electrum>
+
+  @{electrum_exec_path} mr,
+  owner @{HOME}/.electrum/{**,} rwlk,
```
Comment (JeremyRand) [inline contrib/apparmor/usr.local.bin.electrum-nmc:12]: Should be `.electrum-nmc` I think?

File: contrib/apparmor/electrum.appimage [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# Credits : Mikhail Morfikov
```
Comment (JeremyRand) [inline contrib/apparmor/electrum.appimage:1]: This file should be called `electrum-nmc.appimage` I think?

**Comment by robertmin1** [comment]
The modifications have been made, but I've observed that we're also copying the `README.md`, which is leading to errors. Should we consider moving the files to a different folder instead?

**Comment by JeremyRand** [comment]
ACK 93fe42a3479d0e1ada00b8a24a4b2523e2cc6e57. This is difficult for me to test properly, but since upstream merged it, I have no issues with it. I'm going to cherry-pick this to stable branch, and then it can be merged to master in a standard merge from upstream.

**Comment by robertmin1** [comment]
Alright! Squashed the commits

**Comment by JeremyRand** [comment]
Merged to stable branch as https://github.com/namecoin/electrum-nmc/commit/2486d518ca4da1f0721507d61b5335dc360c1201. Closing the PR since we'll pull it into master from upstream. Let me know if anything else is needed for this one.

**Comment by robertmin1** [comment]
Alright!

**Pull request closed without merging**